substrait_consumer/tests/functional/extension_functions/arithmetic_decimal_snapshots/DuckDBConsumer/DuckDBProducer/divide_result.txt

```diff
@@ -0,0 +1,8 @@
+pyarrow.Table
+l_tax: decimal128(15, 2)
+l_extendedprice: decimal128(15, 2)
+DIVIDE_KEY: double
+----
+l_tax: [[0.02,0.06,0.02,0.06,0.04,...,0.06,0.05,0.04,0.00,0.00]]
+l_extendedprice: [[24386.67,58958.28,10210.96,31197.88,31329.60,...,36739.17,16112.40,57084.96,8859.36,1828.91]]
+DIVIDE_KEY: [[8.201201722088339e-7,0.0000010176687650996603,0.000001958679693192413,0.0000019232076025678665,0.0000012767478678310608,...,0.000001633134335914502,0.0000031032000198604804,7.007099593307939e-7,0,0]]
```

Comment:
I think you'll want to add a `round` or a `truncate` or something to any floating point results, because otherwise this is going to be a flaky indicator of changes.

Reply:
or cast to decimal?